client/state/selectors/get-jetpack-credentials-test-status.js

```diff
@@ -0,0 +1,7 @@
+import { get } from 'lodash';
+
+import 'calypso/state/jetpack/init';
+
+export default function getJetpackCredentialsTestStatus( state, siteId, role ) {
+	return get( state, [ 'jetpack', 'credentials', 'testRequestStatus', siteId, role ], 'pending' );
```

Comment:
I'd suggest not introducing any new Lodash usage, as we've been trying to get rid of it - see p4TIVU-9Bf-p2. I believe you should be able to get around this by using optional chaining and nullish coalescing.

Reply:
Good to know, I will work on it.

Reply:
This is now fixed.